src/epfl-mini-card-deck/index.js

```diff
@@ -0,0 +1,96 @@
+import miniCardIcon from './mini-card-icon'
+import './mini-card-panel'
+
+const { __ } = wp.i18n;
+
+const {
+    registerBlockType,
+} = wp.blocks;
+
+const {
+    InspectorControls,
+    InnerBlocks,
+} = wp.editor;
+
+const {
+    PanelBody,
+    ToggleControl,
+    TextControl,
+} = wp.components;
+
+const { Fragment } = wp.element;
+
+const TEMPLATE = [
+	['epfl/mini-card-panel', {}, [] ],
+    ['epfl/mini-card-panel', {}, [] ],
+    ['epfl/mini-card-panel', {}, [] ],
+    ['epfl/mini-card-panel', {}, [] ],
+    ['epfl/mini-card-panel', {}, [] ],
+    ['epfl/mini-card-panel', {}, [] ],
+]
+
+const getAttributes = () => {
+    let atts = {
+        title: {
+            type: 'string'
+        },
+        grayWrapper: {
+            type: 'boolean',
+            default: false,
+        },
+    };
+
+    return atts;
+}
+
+registerBlockType( 'epfl/mini-card-deck', {
+	title: __( 'EPFL Mini Card Deck', 'epfl'),
+	description: 'v1.0.0',
+	icon: miniCardIcon,
+	category: 'common',
+	attributes: getAttributes(),
+	supports : {
+		customClassName: false, // Removes the default field in the inspector that allows you to assign a custom class
+	},
+	edit: ( props ) => {
+        const { attributes, className, setAttributes } = props;
+
+        return (
+            <Fragment>
+                <InspectorControls>
+                    <p><a className="wp-block-help" href={ __('https://www.epfl.ch/campus/services/mini-card-en/', 'epfl') } target="new">{ __('Online help', 'epfl') } </a></p>
+                    <PanelBody title='Format'>
+                        <ToggleControl
+                            label={ __('Wrap with a gray border', 'epfl') }
+                            checked={ attributes.grayWrapper }
+                            onChange={ grayWrapper => setAttributes( { grayWrapper } ) }
+                        />
+                    </PanelBody>
+                </InspectorControls>
+                <div className={ className + ' wp-block-scroll' }>
+                        <h2>{ __('EPFL Mini Card Deck', 'epfl') }</h2>
+                        <TextControl
+                        label={ __('Title (mandatory)', 'epfl') }
```

Comment:
Je suis pour ne pas rendre le titre obligatoire. De plus, j'ai testé et il n'est pas obligatoire actuellement
```suggestion
                        label={ __('Title', 'epfl') }
```